setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='scp-ingest-pipeline',
-    version='1.5.4',
+    version='1.5.6',
```

Comment:
Has there been a change to the metadata convention or convention validation logic since the last PyPI release?  If not, this change seems OK but not strictly necessary, as only a new Docker image (via GitHub release) and not a PyPI release is needed.

Reply:
You're right. I didn't think through making this particular change. I don't plan on a PyPI release with this PR, just an ingest pipeline release for a new docker image.